packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts

```diff
@@ -80,13 +67,9 @@ export class W3mConnectingWcView extends LitElement {
         properties: { message: (error as BaseError)?.message ?? 'Unknown' }
       })
       ConnectionController.setWcError(true)
-      if (CoreHelperUtil.isAllowedRetry(this.lastRetry)) {
```

Comment:
do we still need this util? can we remove?

Reply:
seems `isAllowedRetry` is also used in `AccountController`